drivers/dma/dma_stm32.c

```diff
@@ -516,7 +570,12 @@ static int dma_stm32_stop(struct device *dev, u32_t id)
 		return -EINVAL;
 	}
 
+#ifdef CONFIG_DMAMUX_STM32
+	/* do not disable TC irq */
+#else
```

Comment:
Why not doing ifndef ?

Reply:
This comment I want to add in case of dmamux

Reply:
Maybe you can do:

--
/* do not disable TC irq in case of DMAMUX */
#ifndef CONFIG_DMAMUX_STM32
...
#endif

Reply:
No, this comment is meaningless without DMAMUx

Reply:
Once again, I prefer avoiding empty sections.

Reply:
> Once again, I prefer avoiding empty sections.

@erwango 
I agree